spec.md

```diff
@@ -2274,6 +2374,20 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+// VolumeCondition represents the current health condition of a volume
+message VolumeCondition {
+  option (alpha_message) = true;
+
+  // Indicates the health condition of a volume is not normal.
+  // This field is OPTIONAL. If not present, it indicates the
```

Comment:
Should this field be REQUIRED since it is the first one?

Reply:
Makes sense.  If it is required, we can just use a bool?